wiring/src/spark_wiring_interrupts.cpp

```diff
@@ -107,12 +102,9 @@ bool attachInterrupt(uint16_t pin, raw_interrupt_handler_t handler, InterruptMod
  *******************************************************************************/
 void detachInterrupt(uint16_t pin)
```

Comment:
Let's change this function signature to also return `true`/`false` to propagate the error.

Reply:
Fixed this function signature return type `bool`.